cirq-ionq/cirq_ionq/results.py

```diff
@@ -26,7 +26,9 @@ class QPUResult:
     def __init__(
         self, counts: Dict[int, int], num_qubits: int, measurement_dict: Dict[str, Sequence[int]]
     ):
-        self._counts = counts
+        # We require a consistent ordering, and here we use bitvector as such.
+        # OrderedDict can be removed in python 3.7, where it is part of the contract.
+        self._counts = collections.OrderedDict({k: counts[k] for k in sorted(counts.keys())})
```

Comment:
This will lose ordering in a version of python 3 where python dicts are not ordered, since you construct the OrderedDict from a plain dict. Can use an iterable of tuples instead:
```suggestion
        self._counts = collections.OrderedDict(sorted(counts.items()))
```

Reply:
makes sense, ty!